packages/modal/src/components/modal/Modal.tsx

```diff
@@ -34,6 +38,19 @@ export const Modal: React.FC<ModalProps> = ({
       <Box spacing={activeSpacing} indent={activeIndent}>
         {children}
       </Box>
+      {footer && (
+        <Box
```

Comment:
Replace this `<Box>` with `<ModalFooter>` and move `position`, `bottom` and `boxShadow` down to `ModalFooter`.
Add `sticky` prop to `ModalFooterProps` and pass `stickyFooter` to that prop.

Reply:
```suggestion
        <ModalFooter sticky={stickyFooter}>{footer}</ModalFooter>
```

Reply:
Fixed!